.changeset/poor-boats-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Changed: `secondaryOptions` type
```

Comment:
It's changed and not fixed because we had 2 cases that matched the new type and one that matched the old one.